.circleci/config.yml

```diff
@@ -0,0 +1,111 @@
+aliases:
+  - &restore-cache
+    keys:
+      - dependencies-{{ .Branch }}-{{ checksum "package.json" }}
+      # Fallback in case checksum fails
+      - dependencies-{{ .Branch }}-
+
+  - &save-cache
+    paths:
+      - node_modules
+      - website/node_modules
+    key: dependencies-{{ .Branch }}-{{ checksum "package.json" }}
+
+  - &yarn-install
+    run: |
+      sudo npm i -g yarn@^0.27.5
```

Comment:
There's docs on CircleCI's site: https://circleci.com/docs/2.0/custom-images/

I was going to submit a PR for this, but then I realised it's actually using a bunch of Docker images for various Node.js versions 😛  Initially I was going to do something like this:

```dockerfile
# Dockerfile for building and testing Jest

FROM markhobson/node-chrome
MAINTAINER Daniel Lo Nigro <daniel@dan.cx>

# Yarn repo
ADD https://dl.yarnpkg.com/debian/pubkey.gpg /tmp/yarn-pubkey.gpg
RUN apt-key add /tmp/yarn-pubkey.gpg && rm /tmp/yarn-pubkey.gpg
RUN echo "deb http://dl.yarnpkg.com/debian/ stable main" > /etc/apt/sources.list.d/yarn.list

# Crowdin repo
ADD https://artifacts.crowdin.com/repo/GPG-KEY-crowdin /tmp/crowdin-pubkey.gpg
RUN apt-key add /tmp/crowdin-pubkey.gpg && rm /tmp/crowdin-pubkey.gpg
RUN echo "deb https://artifacts.crowdin.com/repo/deb/ /" > /etc/apt/sources.list.d/crowdin.list

RUN apt-get -y update && \
  apt-get install -y --no-install-recommends \
    yarn \
    default-jre \
    crowdin \
  && \
  apt-get clean && \
  rm -rf /var/lib/apt/lists/*

```